modApiServer/src/org/aion/api/server/types/ArgTxCall.java

```diff
@@ -66,7 +66,7 @@ public ArgTxCall(
     }
 
     public static ArgTxCall fromJSON(
-            final JSONObject _jsonObj, NrgOracle oracle, long defaultNrgLimit) {
+            final JSONObject _jsonObj, long defaultNrgPrice, long txNrg, long contractCreateNrg) {
```

Comment:
I also feel that these values should be constants instead of method parameters.

Reply:
Agree!

Reply:
Done